src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/DumplingHelper.py

```diff
@@ -16,10 +17,19 @@ def install_dumpling():
       url = "https://dumpling.azurewebsites.net/api/client/dumpling.py"
       scriptPath = os.path.dirname(os.path.realpath(__file__))
       downloadLocation = scriptPath + "/dumpling.py"
-      urllib.urlretrieve(url, downloadLocation)
-      subprocess.call([sys.executable, downloadLocation, "install", "--update"])
+      response = urllib2.urlopen(url)
+      if response.getcode() == 200:
```

Comment:
I can't tell from docs, but it seems like urllib2.urlopen ought to throw when the code is not a success code (2xx). I guess it wasn't?
Anyway this seems fine as far as I can tell

Reply:
It should be the case but there was nothing in documentation about on which error code it urlopen raises an exception that's y I added status code check to make it more robust